app/controllers/lead_providers/report_schools/base_controller.rb

```diff
@@ -13,7 +13,11 @@ class BaseController < ::LeadProviders::BaseController
       def start
         clean_form!
 
-        report_schools_form.cohort_id = Cohort.current.id
+        report_schools_form.cohort_id = if FeatureFlag.active?(:multiple_cohorts)
```

Comment:
I think after my suggestion this can always be `Cohort.next`

Reply:
I removed the conditional.